### PR TITLE
spec: required scriptlets dependencies and do not produce errors

### PR DESCRIPTION
**Pull request by pbrezina**
Resolves:
* https://github.com/pbrezina/authselect/issues/96
* https://github.com/pbrezina/authselect/issues/97

### CODE REVIEW

File: rpm/authselect.spec.in [hunk 1]
```diff
@@ -35,6 +35,16 @@ supported by authselect.
...
 Summary: Utility library used by the authselect tool
+# Required by scriptlets
+Requires: %{_bindir}/awk
+Requires: %{_bindir}/cp
+Requires: %{_bindir}/echo
+Requires: %{_bindir}/find
+Requires: %{_bindir}/grep
+Requires: %{_bindir}/rm
+Requires: %{_bindir}/sed
+Requires: %{_bindir}/systemctl
```
Comment (jhrozek) [inline rpm/authselect.spec.in]: I think this is incorrect, shouldn't you require systemd? Also, I can't imagine a system where systemctl does not exist, is this even necessary?

File: rpm/authselect.spec.in [hunk 2]
```diff
@@ -35,6 +35,16 @@ supported by authselect.
 
 %package libs
 Summary: Utility library used by the authselect tool
+# Required by scriptlets
+Requires: %{_bindir}/awk
```
Comment (jhrozek) [inline rpm/authselect.spec.in]: I don't think any of these are needed. At least in the past there was a number of packages that were guaranteed to be in a buildroot, did this change?

**Comment by pbrezina** [comment]
@jhrozek It is based on [this bug](https://bugzilla.redhat.com/show_bug.cgi?id=1630896) where coreutils were not installed.

**Comment by jhrozek** [comment]
Indeed, coreutils is apparently no longer guaranteed to be installed. Can you switch to requiring coreutils, though? Resolving the paths is quite expensive..

**Comment by pbrezina** [comment]
@jhrozek Good to know. See the new patch.

**Comment by pbrezina** [comment]
Pushed to master:
* 49ce5951b0ff9d7bf024f41b9c413640125cc05d
* 7fa4ed9468aa2f078e612ce0c4c0dd7fd2f3875d

**Pull request closed without merging**